ambari/datadog_checks/ambari/ambari.py

```diff
@@ -5,6 +5,10 @@
 from six.moves.urllib.parse import quote, urljoin
 
 
+CLUSTERS_URL = "http://{ambari_server}:{ambari_port}/api/v1/clusters"
```

Comment:
Will this be an issue if the user places a trailing slash in the URL in the config?